source/common/common/packed_struct.h

```diff
@@ -85,8 +85,8 @@ template <class T, uint8_t max_size, class ElementName> class PackedStruct {
     static_assert(std::is_enum_v<ElementName>);
     static_assert(max_size > 0);
 
-    RELEASE_ASSERT(a_capacity <= max_size,
```

Comment:
I think this will be a UB if a_capacity is larger than max_size and looking at the code it is possible to make it happen without full test coverage (i.e. have no tests that set all possible timeouts at the same time).

This one definitely needs a TODO and an urgent fix to this class.